src/SlackNotificationRouterChannel.php

```diff
@@ -38,6 +38,8 @@ public function send($notifiable, Notification $notification)
     {
         $route = $notifiable->routeNotificationFor('slack', $notification);
 
+        if ($route === false) return null;
```

Comment:
```suggestion
        if ($route === false) {
            return null;
        }
```

Reply:
Changed the format and also added a test case.